packages/web/src/web-tracer.js

```diff
@@ -71,20 +73,55 @@ function tag(key, value) {
   }
 }
 
+function handleLogLevel(_logLevel) {
+  let logLevel;
+  switch (_logLevel) {
+    case 'ALL':
+      logLevel = DiagLogLevel.ALL;
+      break
+    case 'DEBUG':
+      logLevel = DiagLogLevel.DEBUG;
+      break
+    case 'INFO':
+      logLevel = DiagLogLevel.INFO;
+      break
+    case 'WARN':
+      logLevel = DiagLogLevel.WARN
+      break
+    case 'ERROR':
+      logLevel = DiagLogLevel.ERROR
+      break
+      // Default is Open Telemetry default which is DiagLogLevel.INFO
+    default:
+      return
+  }
+  diag.setLogger(new DiagConsoleLogger(), logLevel)
+}
+
 function init(_configData) {
   const configData = _configData;
+
+  if (configData.logLevel) {
+    handleLogLevel(configData.logLevel)
+  }
+
+  // Epsagon debug overrides configData.logLevel
+  if (configData.epsagonDebug) {
+    diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG)
+  }
+
   if (configData.isEpsagonDisabled) {
-    console.log('epsagon disabled, tracing not running');
+    diag.info('epsagon disabled, tracing not running');
     return undefined;
   }
 
   if (existingTracer && !configData.isTest) {
-    console.log('tracer already initialized, remove duplicate initialization call');
+    diag.info('tracer already initialized, remove duplicate initialization call');
     return undefined;
   }
 
   if (!configData.token) {
-    console.log('Epsagon token must be passed into initialization');
+    diag.error('Epsagon token must be passed into initialization');
```

Comment:
maybe keep this specific one also in console log for an easy start for new implementers

Reply:
fixed